### ISSUE TITLE
[Issue]: Tilt Sensor

### ISSUE THREAD

**Issue by kizzoknight** [sanjay900/guitar-configurator#60]
### Please describe your issue.

When I have a tilt sensor connected its always active when i chage the setting of the sensitvity in clone hero it just doesnt work at all

### Version

yes

### What operating system are you using?

Windows 10

### What microcontroller are you using?

arduino pro micro 5V

### How is your controller programmed?

Wii

**Comment by sanjay900** [comment]
Are you using PS3 mode or xinput

**Comment by kizzoknight** [comment]
xinput

**Comment by kizzoknight** [comment]
it also shows on z rotation but arent tilt switches either on or off so should just be a button press right?

**Comment by sanjay900** [comment]
Nah it should map tilt to an axis, because there what the game expects

You shouldn't need to calibrate anything on clone heros end

**Comment by sanjay900** [comment]
what is it showing on z rotation when you tilt?

**Comment by kizzoknight** [comment]
![image](https://user-images.githubusercontent.com/55956476/207470682-83a52a48-00c4-488b-9dea-c49970b74392.png)
Z rotation is what changed when tilting

**Comment by sanjay900** [comment]
and how is it changing exactly? sounds more like you just need to reset the calibration settings in ch

**Comment by sanjay900** [comment]
also, how do you have your tilt sensor connected

**Comment by kizzoknight** [comment]
ive tried the tilt sensor between GND and an Analogue pin and GND and a digital pin and it jumps from half way on Z rotation to full i will try the calibration in windows soon

**Comment by sanjay900** [comment]
did you tell the tool you had an analog sensor?

**Comment by kizzoknight** [comment]
yes and selected the pin

**Comment by sanjay900** [comment]
try hitting the "invert" button inside of clone hero

**Comment by kizzoknight** [comment]
that makes it not work at all

**Comment by sanjay900** [comment]
so hold on what is it doing now, is it just always active
and then inverting makes it never activate?

**Comment by ruenes1** [comment]
Hello,

I think I'm having the same issue. I use a digital tilt sensor SW-520D and, when it is not titled, the Z rotation axis is all the way up and CH interprets is a start power. Only when I tilt it, then the Z rotation axis is neutral. I'm sure the orientation of the physicial tilt sensor is correct because I connected a led in line, and it behaves as expected (and inversely to the expected behavior of the Z rotation axis). Using the setting to invert tilt into the configuration tool does nothing (as I understand it is ment for analog tilting).

Thanks

**Comment by sanjay900** [comment]
Have you set it to digital tilt in the tool?

**Comment by ruenes1** [comment]
Yes I've set it to digital. But I think I found the issue. Since I've wired a led in line with the switch by following the schematic for the frets, the input pin is receiving vcc rather than gnd when the sensor is tilted, thereby resulting in inversed behavior. Now I've wired it like a standard button without the led and it works as expected. Having the inline led was nice to check when the titling actives/deactivates but I understand the current firmware only supports leds for frets.
